docs/spelling_wordlist.txt

```diff
@@ -389,6 +389,7 @@ Decrypt
 decrypt
 decrypted
 Decrypts
+dedupe
```

Comment:
If we use deduplicate there is no need to add new word to the exclusion list

Reply:
Sure, I'll make the change.